CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 The changelog lists most feature changes between each release. 
 
+## Upcoming release
+- Chore: [update to Dagster 1.10.2.](https://github.com/mobidata-bw/ipl-dagster-pipeline/pull/188)
```

Comment:
Can you eloborate what effects the upgrade will have? Will the automaterialization happen differently in practice?